src/tkn.ts

```diff
@@ -269,20 +274,20 @@ export class Command {
   static createPipelineResource(yamlFile: string): CliCommand {
     return newTknCommand('resource', 'create', '-f', yamlFile);
   }
-  static tknStatus(): CliCommand {
+  static checkTekton(): CliCommand {
     return newK8sCommand('auth', 'can-i', 'create', 'pipeline.tekton.dev', '&&', 'kubectl', 'get', 'pipeline.tekton.dev');
   }
   static updateYaml(fsPath: string): CliCommand {
     return newTknCommand('apply', '-f', fsPath);
   }
   static listTaskRun(): CliCommand {
-    return newTknCommand('taskrun', 'list', '-o', 'json');
+    return newK8sCommand('get', 'taskrun', '-o', 'json');
   }
   static listConditions(): CliCommand {
     return newK8sCommand('get', 'conditions', '-o', 'json');
   }
   static listPipelineRun(): CliCommand {
-    return newTknCommand('pipelinerun', 'list', '-o', 'json');
+    return newK8sCommand('get', 'pipelinerun', '-o', 'json');
```

Comment:
can we show the running pipelineRun and taskRun at the top? I think when we were using tkn command that time we were not facing this problem.

<img width="446" alt="Screenshot 2020-04-17 at 2 31 28 PM" src="https://user-images.githubusercontent.com/9924513/79553417-13a3dd80-80ba-11ea-8a14-624beaae87e3.png">